docs/upgrading.md

```diff
@@ -1,3 +1,10 @@
+### Upgrading errbit from v0.9.0 to v0.10.0
+
+* MongoDB 4.0 is reached EOL at 21 Jun 2018. 4.0 support will be removed after v0.10.0 release.
```

Comment:
“reached EOL on” instead of “is reached EOL at” sorry for nitpick

Reply:
@burnettk this is not nitpicking.

English is not my native language. So, any help with my docs always welcome. :)